### ISSUE TITLE
Pasting does not work in accessibility mode when the experimental edit context is enabled

### ISSUE THREAD

**Issue by aiday-mar** [microsoft/vscode#239342]
- turn on accessibility mode
- turn on the experimental edit context
- pasting does not work 🐛

**Comment by aiday-mar** [comment]
duplicate of https://github.com/microsoft/vscode/issues/239319
